### PR TITLE
counts refresh

### PR DESCRIPTION
**Pull request by JulianWielga**
## Describe your changes

https://github.com/user-attachments/assets/63d2b9f4-8a96-4ddb-8e7d-39cf81ad14bb

https://github.com/user-attachments/assets/61257d76-c77b-4ef1-be0f-8477469f7143

https://github.com/user-attachments/assets/f6368bb8-de86-4f79-9b51-faeec99e258a

## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

**Comment by Dzuming** [comment]
Good job, I like this functionality. I found a small things to consider

1. I selected the autorefresh option but when I opened counts again, the refresh disabled is visible. Could it be possible to display the currently selected refresh value somehow?

https://github.com/user-attachments/assets/a7ec5e93-d9fe-4261-a2e2-a8274c8808e6


2. I'm able to run counts in the fragment when passing query parameters. If it's easy to validate, we probably could block this situation, but it's rather a minor issue

image

File: designer/client/test/VisualizationUrl-test.js [hunk 1]
```diff
@@ -33,4 +33,15 @@ describe("validating url counts extractors", () => {
...
+
+    it("extract duration from string", () => {
+        expect(extractCountParams({ to: "0", refresh: "1s" }).refreshIn).toBe(1);
+        expect(extractCountParams({ to: "0", refresh: "1m" }).refreshIn).toBe(60);
+        expect(extractCountParams({ to: "0", refresh: "1h" }).refreshIn).toBe(3600);
+
+        expect(extractCountParams({ to: "0" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "1000" }).refreshIn).toBe(false);
+        expect(extractCountParams({ to: "0", refresh: "1d" }).refreshIn).toBe(false);
```
Comment (Dzuming) [inline designer/client/test/VisualizationUrl-test.js:45]: "1d" is not a valid refresh value but "24h" (or more) is fine

Comment (JulianWielga) [reply designer/client/test/VisualizationUrl-test.js:45]: this param is just iso part after T. it doesn't have to be bulletproof. Durations like 1d, even 6h, make no sense and i don't think we should care such users. harmless, not dangerous, not worth attention

**Comment by JulianWielga** [comment]
> 1. I selected the autorefresh option but when I opened counts again, the refresh disabled is visible. Could it be possible to display the currently selected refresh value somehow?

counts window has only options for counts action - not the state of fetched/displayed/refreshed counts data.
I don't want 
> 2. I'm able to run counts in the fragment when passing query parameters. If it's easy to validate, we probably could block this situation, but it's rather a minor issue

**Comment by Dzuming** [comment]
> counts window has only options for counts action - not the state of fetched/displayed/refreshed counts data.
I don't want to store this state now. I changed "refresh disabled" to "no refresh" to avoid "state" association.

That makes sense, initially when I clicked through the feature, I had a feeling that it was some issue, that refresh is enabled but I can see this `refresh disabled` information. The change in the text resolved the problem.

**Pull request merged**